### PR TITLE
Add trigger ISO build script

### PR DESCRIPTION
**Pull request by fepitre**
It's an helper script for crontab usage.

### CODE REVIEW

File: build-infra/trigger-iso-build.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            f.write(f"Build-iso r{release} {release}.{timestamp} {timestamp}")
+
+        cmd = [
+            str(f"/usr/local/lib/qubes-builder-github/github-command.py"),
+            "--no-signer-github-command-check",
+            "Build-iso",
+            f"{tmpdir}/command",
+        ]
+
+        subprocess.Popen(cmd)
```
Comment (marmarek) [inline build-infra/trigger-iso-build.py:30]: add .wait() or .communicate()? or use subprocess.run() ?

**Review by marmarek — CHANGES_REQUESTED** [review]
Shouldn't it be installed somewhere? Like in build-* VMs in some directory (/usr/local?). Otherwise it will be just in dom0 in /srv/formulas.

File: build-infra/trigger-iso-build.py (hunk 1 above)
Comment (fepitre) [reply build-infra/trigger-iso-build.py:30]: if I wait for the process it would wait the whole build process, that's what happened locally.

**Comment by fepitre** [comment]
> Shouldn't it be installed somewhere? Like in build-* VMs in some directory (/usr/local?). Otherwise it will be just in dom0 in /srv/formulas.

yes I just dropped it this morning to have it here, then we can adjust and having it build-* VMs.

**Pull request merged**